vizro-core/src/vizro/static/css/variables.css

```diff
@@ -96,7 +96,7 @@
   --text-code-literal: #6bb5fd;
 }
 
-.vizro_light {
+[data-bs-theme="light"] {
```

Comment:
do we need to add :root here?

Reply:
I just added it in, just in case 👍

Reply:
I added it in to try, but the layout was wrong then, because our default is actually the dark theme. So if we add root here, it will have a light theme while the charts are dark.

Reply:
you meant to say default is light ?

Reply:
Vizro's default is the dark theme 😬 Vizro-bootstrap's default is the light theme as this seems to be the more common way of doing it in other bootstrap themes.

Reply:
Will wait with merging until we've decided on the default

Reply:
Dark will be set as the default